packages/@ember/-internals/routing/lib/services/router.ts

```diff
@@ -62,6 +62,12 @@ function cleanURL(url: string, rootURL: string) {
  */
 export default class RouterService extends Service {
   _router!: EmberRouter;
+  get router() {
+    if (!this._router._didSetupRouter) {
```

Comment:
Since `setupRouter` already does this, do we need to check `this._router._didSetupRouter` here (feels vaguely bad to be looking at private props of a private prop...)?